sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 2.10.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+### Dependency Updates
```

Comment:
Please put  'Dependency Updates' to 'Other Changes' category.
Refer here: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md